spec/services/collection_observer_spec.rb

```diff
@@ -109,5 +109,19 @@
         )
       end
     end
+
+    context 'when reviewers are removed from a collection' do
+      let(:collection_after) { collection.dup.tap { |col| col.managers = [collection.reviewers.first] } }
```

Comment:
should this be `col.reviewers = [collection.reviewers.first]` (instead of `col.managers = ...`)?  or am i confused?  (if i'm not off-base, i'm slightly surprised by the `review_access_removed_email` job getting queued, since i can't see from the rest of the code how that'd happen without a reviewer getting removed, hence my uncertainty)

Reply:
yes, typo... I was working on both. Will fix, thanks for catching.